index.html

```diff
@@ -6,6 +6,20 @@
   </head>
   <body>
     <h1>About me (Benjamin Beeksma)</h1>
+
+    <ul>Places I've lived
+      <li>Washburn, Wisconsin</li>
+      <li>Ashland, Wisconsin</li>
+      <li>Fukuoka, Japan</li>
+      <li>New Orleans, Louisiana</li>
+      <li>Iowa City, Iowa</li>
+      <li>(Coming Soon)Cedar Rapids, Iowa</li>
+    </ul>
+
+    <ol>Top ten books
+      <li></li>
```

Comment:
Let's get some of these added in. You have the structure but still docking you a point for not putting something in the li's